components/authz-service/storage/postgres/migration/legacy/iam.go

```diff
@@ -204,16 +204,14 @@ func legacyPolicyFromV1(pol *v1Policy) (*v2Policy, error) {
 	if _, found := v1CfgmgmtPolicies[pol.ID.String()]; found {
 		cfgmgmtStatement := newV2Statement(Allow, "", []string{},
 			[]string{"*"}, []string{"infra:*"})
-		cfgmgmtStatementDeny := newV2Statement(Deny, "", []string{},
-			[]string{"*"}, []string{"infra:ingest:*"})
 		member, err := newV2Member("user:*")
 		if err != nil {
 			return nil, errors.Wrap(err, "format v2 member (cfgmgmt)")
 		}
 		cfgmgmtPolicy, err := newV2Policy(constants_v2.CfgmgmtPolicyID,
 			"[Legacy] Infrastructure Automation Access",
 			Custom, []v2Member{member},
-			[]v2Statement{cfgmgmtStatement, cfgmgmtStatementDeny}, noProjects)
+			[]v2Statement{cfgmgmtStatement}, noProjects)
```

Comment:
We had added a `DENY` in the force-upgrade migration code originally-- here we remove it.